app/models/user.rb

```diff
@@ -3,9 +3,9 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- Hello, You have done a good job. Can you remove the commented code? because it is creating confusion. Also, you need to be presenting clean code because it shows professionalism.